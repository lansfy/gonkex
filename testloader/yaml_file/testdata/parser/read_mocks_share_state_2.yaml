- name: share state for two testscase MUST skip mocks resets after this test
  method: GET
  path: /some/path1
  request: "requestbody"
  mocksParams:
    shareState: true
  mocks:
    someservice:
      strategy: nop

- name: share state for two testscase MUST skip mocks resets before this test
  method: GET
  path: /some/path2
  request: "requestbody"
  mocksParams:
    shareState: true
