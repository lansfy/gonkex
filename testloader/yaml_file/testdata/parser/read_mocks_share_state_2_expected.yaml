- GetName: share state for two testscase MUST skip mocks resets after this test
  GetMethod: GET
  Path: /some/path1
  GetRequest: requestbody
  GetFileName: testdata/parser/read_mocks_share_state_2.yaml
  FirstTestInFile: true
  ServiceMocksParams:
    SkipMocksResetAfterTest: true
  ServiceMocks:
    someservice:
      strategy: nop

- GetName: share state for two testscase MUST skip mocks resets before this test
  GetMethod: GET
  Path: /some/path2
  GetRequest: requestbody
  GetFileName: testdata/parser/read_mocks_share_state_2.yaml
  ServiceMocksParams:
    SkipMocksResetBeforeTest: true
  LastTestInFile: true
